src/components/app-top-bar.js

```diff
@@ -166,20 +166,24 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
     const [, showVoltageLevel, showSubstation] = useSingleLineDiagram();
     // Equipments search bar
     const [equipmentsFound, setEquipmentsFound] = useState([]);
+    const lastSearchTermRef = useRef('');
 
     const searchMatchingEquipments = useCallback(
         (searchTerm) => {
+            lastSearchTermRef.current = searchTerm;
```

Comment:
It should never append because it's called on `onSearchTermChange`... then if we add a character, remove one, even copy/past the same searchTerm doesn't proc this callback. But ok I could check this, never know each cases.